### PR TITLE
Refactoring security 2

### PR DESCRIPTION
**Pull request by yacc**
### 1/ Remove "unsafe-eval" directive
I removed the "unsafe-eval" directive b/c it's unsafe:
- the CSP errors that this addressed were generated only in development (production works wo this directive) b/c of a bug in the webconfig file
- fixed webconfig file, that wasn't taking the `config.devtool = "inline-source-map";` into account when running `ENV=development pnpm build; pnpm start`, b/c webconfig was expecting to see "development" as an arg only not an ENV var.
### 2/ Move nonce for index to its own middleware

### CODE REVIEW

File: server/applyNonce/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  app.use("/:anything", function (req, res, next) {
+    let v = req.params.anything;
+    switch (v) {
+      case "favicon.png":
+      case "favicon.ico":
+      case "styles":
+      case "all":
+      case "images":
+      case "bundle":
```
Comment (sullof) [inline server/applyNonce/index.js:39]: The files & folders that must go next and be read from the static path may be a setting passed by app.js, like
```
applyNonce(app, {
   static: [ 
       "favicon.png",
       "favicon.ico",
       "styles",
       "all",
       "images",
       "bundle"
    ]
});
```
This way we can set it differently for any website.

Comment (yacc) [reply server/applyNonce/index.js:39]: I'll fix this, good suggestion :-)

**Review by sullof — APPROVED** [review]
It looks good to me

**Comment by yacc** [comment]
@sullof Hmmm, I just noticed that I'm getting an error when building in production, it still build and loads, and I don't think ESlint was triggered before the webpack.config fix. 
Screen Shot 2022-09-22 at 6 21 35 PM

But I'll wait until I fix this to merge to main.

**Comment by sullof** [comment]
Is it ready to merge?

**Comment by sullof** [comment]
There is an issue here. The app reload the animated GIFs again and again, consuming bandwidth for nothing. It must be fixed

**Review by sullof — CHANGES_REQUESTED** [review]
The app reloads the animated GIFs forever. There is probably an error in the way they are rendered by React

**Comment by yacc** [comment]
@sullof  the constant reload is not caused by the GIFs (they don't get downloaded continuously from s3). You can check the network tab and you'll see that they're downloaded only ounce. I also removed the GIFs from the page and I'm still seeing that the page seems to be always loading. I'm investigating why.

**Comment by yacc** [comment]
I did a few test and realized that the loading icon on the tab was running because the page couldn't load the favicon...
Screen Shot 2022-09-28 at 3 20 26 PM
I fixed the CSP rules and now it's fine.
Good catch

**Pull request merged**